content/conditions/earache/main-content-2.md

```diff
@@ -0,0 +1,63 @@
+## Your pharmacist can help with earaches
+
+They can give advice and recommend treatments, such as eardrops.
+
+!!! info_compact
+[Find a pharmacy]({{ findersBaseUrl }}finders/find-help)
+!!!
+
+!!! attention
+## See a GP if:
```

Comment:
Yep, that's now the correct pattern. There was [a change](https://github.com/nhsuk/betahealth/commit/d880425e91518477455dd45d9552d90a6633754b) whilst you were away to replace them all.